test/jdk/javax/net/ssl/sanity/ciphersuites/TLSCipherSuitesOrder.java

```diff
@@ -67,6 +70,10 @@ public static void main(String[] args) {
 
     private TLSCipherSuitesOrder(String protocol, String[] clientcipherSuites,
             String[] servercipherSuites) {
+        // Re-enable protocol if it is disabled.
+        if (protocol.equals("TLSv1") || protocol.equals("TLSv1.1")) {
+            SecurityUtils.removeFromDisabledTlsAlgs(protocol);
+        }
```

Comment:
There are several property updates in method other than the main() method.  It should be fine as the constructor is private.  It is safer to make the update in the main() method as the property is loaded one time only in JVM.

Reply:
I think it is safe to set the property in the constructor because the test always runs in its own VM using the jtreg `othervm` option and there are no methods called by main before the constructor that would cause the value of "jdk.tls.disabledAlgorithms" to be fetched and cached. Let me know if you think otherwise.